.github/ISSUE_TEMPLATE/BUG.md

```diff
@@ -0,0 +1,47 @@
+---
+name: 🐞 Bug Report
+about: Something went awry and you'd like to tell us about it.
+
+---
+
+<!--
+  ⚡️ katchow! We 💛 issues.
+
+  Please - do not - remove this template.
+  Please - do not - skip or remove parts of this template.
+  Or your issue may be closed.
+
+  👉🏽 Need help or tech support? Please don't open an issue!
+  Head to https://gitter.im/rollup/rollup or https://stackoverflow.com/questions/tagged/rollupjs
+
+  ❤️ Rollup? Please consider supporting our collective:
+  👉 https://opencollective.com/rollup/donate
+-->
+
+- Rollup Version:
```

Comment:
As this is for rollup-plugin-node-resolve, the plugin version would also be very important! How about

```
Node-Resolve Plugin Version:
Rollup Version:
Operating System (or Browser):
Node Version:
```